opscenter/README.md

```diff
@@ -0,0 +1,89 @@
+Setting up an OpsCenter
+===
+An OpsCenter is a service that you can setup to provide a distribution point for your applications and dependant applications. Depending on how you choose to configure the service, it can retrieve updates from Gravitational's OpsCenter.
+
+Pre-requisites
+---
+ - Login for [https://get.gravitational.io](https://get.gravitational.io)
+ - Telekube binaries
+
+Create an SSL certificate
+---
+The OpsCenter will use SSL to secure connections to it. In this repository are the files `example-server.crt` and `example-server.key` which hold the certificate and key to a self-signed certificate for `example.gravitational.io`. Do deploy your own OpsCenter, you should create a certificate for your server. We recommend using a certificate signed by a trusted root authority, but you can use self-signed certificates if you wish. Here's  some information on [how to create your own self-signed certificate](http://www.akadia.com/services/ssh_test_certificate.html).
```

Comment:
I think using self signed certs should be a separate section, should stay out of the main flow for the reader (place it below)